tests/component/podcasts/test.httpRequests.spec.js

```diff
@@ -0,0 +1,83 @@
+const supertest = require('supertest')
+const { mockIllegalFieldsPodcast, mockPodcast, mockNewPodcast, mockExtraFieldsPodcast } = require('./mockPodcast.js')
+
+const app = require('../../../app')
+
+jest.mock('../../../models/fileModel', () => require('./mockFileModel.js'))
+
+describe('Podcast component test', () => {
+  describe('Add podcast tests', () => {
+    it('should return 200 when podcast is added with all valid fields', async () => {
+      await supertest(app)
+        .post('/podcast/new')
+        .send(mockNewPodcast)
+        .expect(200)
+    })
+    it('should return 400 when trying to add podcast with illegal fields', async () => {
+      await supertest(app)
+        .post('/podcast/new')
+        .send(mockIllegalFieldsPodcast)
+        .expect(400)
+    })
+    it('should return 400 when trying to add podcast extra fields', async () => {
+      await supertest(app)
+        .post('/podcast/new')
+        .send(mockExtraFieldsPodcast)
+        .expect(400)
+    })
```

Comment:
it should also return 400 if a bad data was passed, no? for example if in field numberOfEpisodes we are passing a string. this test is missing